app/models/poem.rb

```diff
@@ -1,5 +1,8 @@
 class Poem < ApplicationRecord
+  POEM_STATUSES = ["Rough Draft", "Ready", "Submitted", "Accepted", "Published"]
```

Comment:
Not just the array, but also the string. I sometimes see `.map(&:freeze).freeze`.